docs/beamrunner/beamrunner-3-sample.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: docs
+title:  Sample application for IBM® Streams Runner for Apache Beam
+navtitle: Sample application
+description:  description
+weight:  10
+published: true
+tag: beam
+prev:
+  file: beamrunner-2-install
+  title: Installing
+next:
+  file: beamrunner-4-monitor
+  title: Monitoring
+---
+
+The `TemperatureSample` application takes temperature readings from multiple devices. The application splits the readings into "good" (valid) and "bad" (invalid) readings based on a specific threshold. It counts the bad readings and generates some basic statistics for the good readings, and finally logs the results. The full source is available in the `$STREAMS\_RUNNER\_HOME/samples/src/com/ibm/streams/beam/sample/temperature` directory, but excerpts are included here to describe the structure of the pipeline.
+
+The devices are artificial; they use the Beam CountingInput transforms named Counter\_*n* to drive another transform named Device\_*n* to generate random readings:
+

```

Comment:
@johnmac-ibm @pgerv12 @pamlane I think we should re-order this page. DIscussing the application in depth here might lose a newbie. After briefly describing what it does as in the paragraph above, I would put the command to run it first, so they can see it doing something.  Then show them either a high level overview of the application either using the application graph itself,  or an image similar to this in the Beam wordcount (https://beam.apache.org/images/wordcount-pipeline.png)

That gives them a rough idea of what is to be done before walking through the code.